shared/chat/conversation/messages/shared-styles.tsx

```diff
@@ -1,46 +1,45 @@
 import * as Styles from '../../../styles'
 
-const editing = {
```

Comment:
its never safe to pull colors at the module level it has to be wrapped in a function that gets called else your light/dark will be incorrect